install/kubernetes/cilium/values.yaml.tmpl

```
@@ -52,6 +52,16 @@ serviceAccounts:
     name: cilium
     automount: true
     annotations: {}
+  nodeinit:
+    create: false
```

Actually, I think this should default to true since `enabled` is false.

```suggestion
    create: true
```